portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+
+/*
+*  Servlet that adds comments
+*/
+@WebServlet("/handle-comments")
+public class DataServlet extends HttpServlet {
+
+  private ArrayList<String> messages;
+
+  @Override
+  public void init() {
+    messages = new ArrayList<>();
+  }
+  
+  
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      String text = request.getParameter("add-content");
+      System.out.println(text);
+      messages.add(text);
+      response.sendRedirect("comment.html");
+  }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException { 
+    String json = convertToJson(messages);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+   private String convertToJson (ArrayList list) {
+    Gson gson = new Gson();
+    String json = gson.toJson(list);
+    return json;
```

Comment:
Rather than returning the result directly, it would be nice to do some basic formatting here. Right now, all the inputs are put in a single line comma separated string. Maybe adding a new line between each comment would be enough for it to look a bit nicer.